front/internals/webpack/webpack.config.js

```diff
@@ -150,58 +153,64 @@ const config = {
     }),
 
     new ForkTsCheckerWebpackPlugin({
-      checkSyntacticErrors: true,
-      tsconfig: path.join(process.cwd(), 'app/tsconfig.json'),
-      silent: !!argv.json, // silent when trying to profile the chunks sizes
+      async: isDev,
+      typescript: {
+        configFile: path.join(process.cwd(), 'app/tsconfig.json'),
+      },
+      logger: { infrastructure: !!argv.json ? 'silent' : 'console' }, // silent when trying to profile the chunks sizes
     }),
 
     new CleanWebpackPlugin(),
 
     new HtmlWebpackPlugin({
       template: 'app/index.html',
       templateParameters: {
-        GOOGLE_MAPS_API_KEY: process.env.GOOGLE_MAPS_API_KEY
-      }
+        GOOGLE_MAPS_API_KEY: process.env.GOOGLE_MAPS_API_KEY,
+      },
     }),
 
     // new BundleAnalyzerPlugin(),
 
     // new webpack.ProgressPlugin(),
 
-
-
     // remove all moment locales except 'en' and the ones defined in appLocalesMomentPairs
-    !isDev && new MomentLocalesPlugin({
-      localesToKeep: [...new Set(Object.values(appLocalesMomentPairs))]
-    }),
-
-    !isDev && new MomentTimezoneDataPlugin({
-      startYear: 2014,
-      endYear: currentYear + 8,
-    }),
-
-    !isDev && new MiniCssExtractPlugin({
-      filename: '[name].[contenthash].css',
-      chunkFilename: '[name].[contenthash].chunk.css'
-    }),
-
-    !isDev && new webpack.HashedModuleIdsPlugin(),
```

Comment:
I'm wondering the reason to remove the module id hashing, could you let me know ?

Reply:
With webpack 5 there is no need for an additional plugin for this, it's part of the optimization settings: https://webpack.js.org/migrate/5/#update-outdated-options
I though I added it but I don't see it now (I shuffled things quite a bit at some point) so thanks for spotting. Should be there now (+ now it's called `deterministic` not `hashed`).